### DISCUSSION TITLE
Image Tag No Height Member Error

### DISCUSSION THREAD

**Discussion by nonfungibletunji** [elementary-swift/elementary#37] [Q&A] [answered]
.src("ttps://doodleipsum.com/700x700/outline?bg=3b5bdb"), .height("800"),
    |                                                                                `- error: type 'HTMLAttribute<HTMLTag.img>' has no member 'height'

image tag

**Comment by nonfungibletunji** [comment]
#35

**Reply by sliemeobn** [reply, marked as answer]
there is now #42
